samples/bot-tab-conversations/csharp/README.md

```diff
@@ -156,7 +156,7 @@ The tour files can be found in the `.tours` directory.
     * Set the 'Bot endpoint address' to `https://<randomsubdomain>.ngrok.io/api/messages`.
     * Create a client secret, being sure to copy the secret for replacing in the `appSettings.json` below.
     * *Note: if you restart Ngrok you may have to update the messaging endpoint domain URL you have set in your Bot Configuration*
-* [Update the AAD App to enable Teams SSO](https://docs.microsoft.com/en-us/microsoftteams/platform/tabs/how-to/authentication/auth-aad-sso?tabs=dotnet#develop-an-sso-microsoft-teams-tab)
+* [Update the AAD App to enable Teams SSO](https://docs.microsoft.com/en-gb/microsoftteams/platform/tabs/how-to/authentication/tab-sso-register-aad)
```

Comment:
why en-gb?

Reply:
I am just a big fan of the Queen's English. Replaced